SwiftCompilerSources/Sources/SIL/Type.swift

```diff
@@ -214,6 +215,16 @@ public struct Type : TypeProperties, CustomStringConvertible, NoReflectionChildr
     }
     return false
   }
+
+  public func getEnumCasePayload(caseIdx: Int, function: Function) -> Type {
+    bridged.getEnumCasePayload(caseIdx, function.bridged).type
+  }
+
+  public func mapTypeOutOfContext() -> Type { bridged.mapTypeOutOfContext().type }
```

Comment:
Please implement this using `AST.Type.mapTypeOutOfContext` and not with bridging